WalletWasabi.Gui/UiConfig.cs

```diff
@@ -167,5 +169,13 @@ public bool RunOnSystemStartup
 		[JsonProperty(PropertyName = "HistoryTabViewSortingPreference")]
 		[JsonConverter(typeof(SortingPreferenceJsonConverter))]
 		public SortingPreference HistoryTabViewSortingPreference { get; internal set; } = new SortingPreference(SortOrder.Decreasing, "Date");
+
+		[DefaultValue(false)]
+		[JsonProperty(PropertyName = "HideOnClose", DefaultValueHandling = DefaultValueHandling.Populate)]
```

Comment:
I wonder whether it is possible/useful to do

```suggestion
		[JsonProperty(PropertyName = nameof(HideOnClose), DefaultValueHandling = DefaultValueHandling.Populate)]
```

here.